lib/minimap.js

```diff
@@ -860,7 +860,14 @@ export default class Minimap {
    * @returns {number} The devicePixelRatio in the Minimap
    */
   getDevicePixelRatio() {
-    return this.getDevicePixelRatioRounding() ? Math.floor(devicePixelRatio) : devicePixelRatio
+    if (this.getDevicePixelRatioRounding()) {
+      if (devicePixelRatio >= 0.5) {
+        return Math.round(devicePixelRatio)
+      } else {
+        return Math.round(devicePixelRatio * 10) / 10
```

Comment:
This would fail if `devicePixelRatio` is `0.049` (quite theoretical value, I know). Why not simply return the whole `devicePixelRatio`?

What is the benefit of having a rounded floating value instead of the full one? Why `0.8` is better than `0.8958333134651184`?

Why not simply:

```js
    return (devicePixelRatio >= 0.5) ? Math.round(devicePixelRatio) : devicePixelRatio
```

Reply:
> This would fail if devicePixelRatio is 0.049 (quite theoretical value, I know). Why not simply return the whole devicePixelRatio?


That's impossible. That would be a screen that is like a needle😄 

> What is the benefit of having a rounded floating value instead of the full one? Why 0.8 is better than 0.8958333134651184?

This makes all the calculations much faster as the engine would not need to calculate all the pixels accurately.

Reply:
https://github.com/atom-minimap/minimap/pull/799/commits/3c506862292a43d8ae0ecf4169ae642411a22080

Reply:
https://en.wikipedia.org/wiki/Fixed-point_arithmetic